linode/manager
# upcoming: [M3-7704] - Disable Cloning, Private IP, Backups for edge regions

## Description 📝
Cloning, Private IP, Backups, Lish Console via SSH, and VPC will not be supported for edge regions so we want to disable them.

## Changes  🔄
- Disabled the Backups and Private IP checkbox in the Linode Create page when an edge region is selected and display a warning notice
- Disabled Private IP radio in the "Add an IP Address" drawer (Linode details page -> Network tab -> Add An IP Address) for Linodes in an edge region with a warning notice
- Disabled the "Enable Backups" button in the Backups tab of the Linode details page with a warning notice
- Disabled the "Clone" action in the Linode landing dropdown action menu and Linode entity detail action menu with a tooltip explanation
- Updated LISH Console via SSH text in Linode Entity Detail Header to N/A

## Preview 📷

| ![image](https://github.com/linode/manager/assets/115299789/49f4891f-6085-4127-bf13-2988b5bc7f55) | ![image](https://github.com/linode/manager/assets/115299789/a55d1f87-ddab-4904-98fa-e23baef22ca2) |
| ------- | ------- |
| ![image](https://github.com/linode/manager/assets/115299789/073e1348-848c-473d-a552-ca47612d4da0) | ![image](https://github.com/linode/manager/assets/115299789/f80bae95-3b52-4c7a-9012-6285c684f8e4) |
| ![image](https://github.com/linode/manager/assets/115299789/ac953328-346f-4137-8a57-fd6230e801ae) |

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Turn on the MSW

### Verification steps
(How to verify changes)
- See the Changes section

```
yarn test AddonsPanel
yarn test BackupsPlaceholder
yarn test AddIPDrawer
yarn test LinodeActionMenu
```

## As an Author I have considered 🤔

*Check all that apply*
